tools/setup_helpers/generate_code.py

```diff
@@ -84,10 +84,17 @@ def generate_code(ninja_global=None):
         if not os.path.exists(d):
             os.mkdir(d)
     gen_autograd(
+<<<<<<< HEAD
```

Comment:
these changes are local to fbcode and shouldn't be synced back

Reply:
@orionr